README.md

```diff
@@ -95,9 +95,23 @@ final FiksIOKonfigurasjon fiksIOKonfigurasjon = FiksIOKonfigurasjon.defaultProdC
         .keyStorePassword(keyStorePassword)
         .build());
 ```
+KontoKonfigurasjon.builder() kan ta inn flere private nøkler. De kan legges til en og en eller som en liste:
+```java
+KontoKonfigurasjon.builder()
+        .kontoId(new KontoId(kontoId))
+        .privatNokkel(privateKey1)
+        .privatNokkel(privateKey2)
+        .build(),
+```
+```java
+KontoKonfigurasjon.builder()
+        .kontoId(new KontoId(kontoId))
+        .privateNokler(Arrays.asList(privateKey1, privateKey2))
+        .privatNokkel(privateKey2)
```

Comment:
Sluttresultat etter dette er en samling nøkler ala dette under, ikke sant?
privateNokler = [privateKey1, privateKey2, privateKey2]

Kan gjerne rename siste nøkkel til *3 slik at man får følgende:
privateNokler = [privateKey1, privateKey2, privateKey3]


PS: Ellers er det vel ingen unikhetssjekk på nøkler, så førstnevnte konfigurasjon er vel gyldig, er den ikke?

Reply:
Ja, det stemmer. For klarhets skyld kan jeg kalle den siste for 3, men det er riktig som du sier at det er en gyldig konfigurasjon med samme nøkkel